cmd/protoc-gen-go-grpc/main.go

```diff
@@ -32,13 +32,23 @@ package main
 
 import (
 	"flag"
+	"fmt"
+	"google.golang.org/grpc"
```

Comment:
Put non-standard package imports in the second/third grouping. By convention, the first package grouping is for standard packages.

Reply:
Good catch! Fixing it now.